### PR TITLE
[Ready] Add api reference

### PR DESCRIPTION
**Pull request by Yassir4**
Add wikis for API reference

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,9 +87,23 @@ const TopBar = ({ play, fullScreen }) => (
...
-// TODO : https://github.com/barmej/react-native-youtube-player/blob/master/src/mobile/types.tsx#L54
-
+|   Property    |      Type     |  description  |
+| :---          | :---:          |:---          |
+| videoId(required)| string     | youtube video Id  |
+| autoPlay      |  Boolean      | auto play the video |
+  onError       |  function() => void   |execute a function on error     |
+| loop(required)| Boolean   | loop the video|
+| style         | object        | You can pass this to override some default styles |
+| topBar        | function(play: boolean, fullScreen: boolean)  | function which takes an object with the play and fullScreen status and return a react element to be used as a topBar   |
```
Comment (yjose) [inline README.md]:
```suggestion
| topBar        | (play: boolean, fullScreen: boolean) => React.ReactNode  | function which takes  the play and fullScreen status and return a react element to be used as a topBar   |
```

File: README.md [hunk 2]
```diff
@@ -87,9 +87,23 @@ const TopBar = ({ play, fullScreen }) => (
...
+| showFullScreenButton| Boolean |   display a button to allow user to see the video on fullScreen     |
+| onFullScreen  | function(fullScreen: Boolean) | execute a function in fullScreen |
+| onStart       | function() => void    | execute a function on start |
```
Comment (yjose) [inline README.md]:
```suggestion
| onStart       | () => void    | execute a function on start |
```

Comment (yjose) [reply README.md]: same for others

File: README.md [hunk 3]
```diff
@@ -87,10 +87,22 @@ const TopBar = ({ play, fullScreen }) => (
 ```
 
 ## API reference
+|   Property    |      Type     |  description  |
+| :---          | :---:          |:---          |
+| videoId(required)| string     | Youtube video Id  |
+| autoPlay      |  Boolean      | Auto play the video |
+  onError       |  () => void   |Execute a function on error     |
+| loop(required)| Boolean   | Loop the video|
```
Comment (yjose) [inline README.md]: `loop` not required 😄

Comment (Yassir4) [reply README.md]: 😅

**Pull request merged**